website/content/specification/syntax/constraints.md

```diff
@@ -202,6 +202,20 @@ The `index-has-key` constraint is a type of Metaschema constraint that cross-ref
 The `@name` flag of an `<index>` constraint MUST specify the name of a previously defined `index` constraint.
 
 The `index-has-key` constraint has the same flags and assemblies as a [`index`](#index-constraints) constraint.
+## `is-unique` constraints
+
+The `is-unique` constraint is a type of Metaschema constraint that checks that a computed key, based on field and flag values, does not occur more than once.
```

Comment:
Blocking: It took a little while to understand why have `is-unique` at all so maybe we want should editorialize a little here. I would like others' feedback here as well. I think it would be very helpful to say up front why it is basically like `index` but different: it operates the same without explicitly building an index.

```suggestion
The `is-unique` constraint is a type of Metaschema constraint that checks that a computed key, based on field and flag values, does not occur more than once. Unlike `index`, an explicit, named index is not created. Therefore, this constraint MUST not define a `@name` attribute.
```